.github/workflows/budibase_ci.yml

```diff
@@ -44,9 +44,31 @@ jobs:
           node-version: 14.x
           cache: "yarn"
       - run: yarn
-      - run: yarn build
+      - run: yarn build --projects=@budibase/string-templates
+      - run: yarn nx run-many -t=check:types
+
+  test-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+          cache: "yarn"
+      - run: yarn
+      - run: yarn build --projects @budibase/shared-core,@budibase/string-templates
+      - run: yarn test --ignore=@budibase/worker --ignore=@budibase/server --ignore=@budibase/pro
+      - uses: codecov/codecov-action@v3
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }} # not required for public repos
+          name: codecov-umbrella
+          verbose: true
 
-  test:
+  test-apps:
```

Comment:
I do worry the name of this could be confusing - everytime I see it I assume its just the `server` package being tested due to it generally being called the `apps` service!

Reply:
This "apps" refer to the apps vs packages in monorepos. What do you think would be the best name to refer to the "deployables" vs npm-like packages?

Reply:
I feel like `services` is a better name - suggesting it is actual runnable services, rather than libraries.

Reply:
+1 on `services`